VideoLocker/src/main/java/org/edx/mobile/view/CourseBaseActivity.java

```diff
@@ -134,7 +105,6 @@ protected void onDestroy() {
         isDestroyed = true;
```

Comment:
Looks like I meant to use it to check the status of the Fragment before delivering the result from the `GetCourseStructureTask`, but somehow I missed actually implementing it: https://github.com/edx/edx-app-android/pull/360#discussion_r39686305

You can use it for that purpose if you want.